appyters/Achilles_Harmonizome_ETL/Achilles.ipynb

```diff
@@ -92,6 +84,51 @@
                 "# Initialization"
             ]
         },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "%%appyter code_eval\n",
+                "\n",
+                "{% do SectionField(\n",
+                "    name='data',\n",
+                "    title='Upload Data',\n",
+                "    img='load_icon.png'\n",
+                ") %}\n",
+                "\n",
+                "{% do DescriptionField(\n",
+                "    name='description',\n",
+                "    text='The examples below were sourced from <a href= \"https://depmap.org/portal/download/\" target=\"_blank\">depmap.org/portal/download</a>. If clicking on the examples does not work, they should be downloaded directly from the source website.',\n",
+                "    section='data'\n",
+                ") %}\n",
+                "\n",
+                "{% set matrix_file = FileField(\n",
+                "    constraint='.*\\.gct$',\n",
+                "    name='rnai',\n",
+                "    label='RNAi Screen (gct)',\n",
+                "    default='Input/Achilles/Achilles_v2.4.6.rnai.gct',\n",
+                "    examples={\n",
+                "        'Achilles_v2.4.6.rnai.gct': \n",
+                "        'https://depmap.org/portal/download/api/download/external?file_name=15%2FAchilles_v2.4.6.rnai.gct'\n",
+                "    },\n",
+                "    section='data'\n",
+                ") %}\n",
+                "\n",
+                "{% set sample_file = FileField(\n",
+                "    constraint='.*\\.txt$',\n",
+                "    name='sample_info',\n",
+                "    label='Sample Info (txt)',\n",
+                "    default='Input/Achilles/Achilles_v2.4.6_v2.19.1_v2.20.1_SampleInfo.txt',\n",
+                "    examples={\n",
+                "        'Achilles_v2.4.6_v2.19.1_v2.20.1_SampleInfo.txt':\n",
+                "        'https://depmap.org/portal/download/api/download/external?file_name=15%2FAchilles_v2.4.6_v2.19.1_v2.20.1_SampleInfo.txt'\n",
```

Comment:
What happens if the download fails because some form of logging in or authorization is required? Should I just omit those examples

Reply:
Removing the directory structure works if that works for you. With respect to the authorization issue, this is a challenge--on one hand we want users to be able to easily run an example, on the other hand some files require manual intervention to actually resolve.

Unfortunately, the way the examples works currently is by downloading things server-side instead of directing the user to download it, in which case a link which requires authentication will most certainly be broken. In these cases, I may be able to detect it and
1. In tests, report a warning but don't continue the test (it will still be considered a success), in these cases we naturally cannot run the notebook without user interaction, which is okay.
2. In production, in the case of errors during download direct the user to the website--this way they will see any authentication or 404 style errors without having to deal with not knowing what went wrong.

No further action should be required on your part, when I have the above things working I will merge your PRs.

Reply:
Got it. Thanks for your help.